### ISSUE TITLE
Compile for Apple M1 macs?

### ISSUE THREAD

**Issue by Altonss** [texstudio-org/texstudio#1964]
### Describe the feature and the current behavior/state
Is there a plan to compile the macOs version Texstudio for apple silicon/arm architecture?

### Who will benefit with this feature?
Users of apple m1 macs

**Comment by Altonss** [comment]
Duplicate of #1641

**Comment by BrushXue** [comment]
I'm compiling it myself by running `BUILD.sh`. Usually it takes 5-6 minutes on M1.

**Comment by BJKle** [comment]
> I'm compiling it myself by running `BUILD.sh`. Usually it takes 5-6 minutes on M1.

How can i build a .dmg to distribute it to another M1 machine?
Even on the machine I built it the app already crashes with " EXC_BAD_ACCESS (SIGKILL (Code Signature Invalid))" after I did run "macdeployqt texstudio.app -dmg".

(I'm not a developer and I just followed the instructions from the wiki.)
It would be really beneficial to provide at least a universal app.
